dbt/adapters/redshift/relation_configs/materialized_view.py

```diff
@@ -122,8 +122,8 @@ def parse_model_node(cls, model_node: ModelNode) -> dict:
             "mv_name": model_node.identifier,
             "schema_name": model_node.schema,
             "database_name": model_node.database,
-            "backup": model_node.config.get("backup"),
-            "autorefresh": model_node.config.get("auto_refresh"),
+            "backup": model_node.config.extra.get("backup"),
+            "autorefresh": model_node.config.extra.get("auto_refresh"),
```

Comment:
should this be `auto_refresh`?

Reply:
I went back and forth on it. So Redshift doesn't use any spaces anywhere, and I've been trying to adopt the concept that adatper-specific settings should appear in adapter-specific vernacular. We'll always use `schema_name` regardless of what the database calls it because we need it to be consistent. But if Redshift decided to name it `autorefresh`, I'm going with that on the object. That being said, dbt has adopted the naming convention of snake case for everything. So we will accept it in the config as `auto_refresh` and then translate it to `autorefresh` to put it on the object.

tl;dr; No, I don't think so.

Reply:
fantastic thanks for the clear up makes complete sense